AppCenterCrashes/AppCenterCrashes/MSACCrashes.h

```diff
@@ -148,16 +149,14 @@ typedef NS_ENUM(NSUInteger, MSACUserConfirmation) {
  *
  * @see MSACCrashesDelegate
  */
-+ (void)setDelegate:(_Nullable id<MSACCrashesDelegate>)delegate;
+@property(class, nonatomic) id<MSACCrashesDelegate> _Nullable delegate;
```

Comment:
Worth adding `weak` here? At least for explicitly mentioning it and lift customer's doubt and for  consistency with the `MSACDistribute` delegate which is declared weak.